.git-blame-ignore-revs

```diff
@@ -147,3 +147,6 @@ ff5c8f6cc3d1f2e017e86d50965c14b71f00567b
 
 # pico-sdk: switch to finalAttrs (#329438)
 8946018b0391ae594d167f1e58497b18de068968
+
+# nixos/ollama: format with nixfmt-rfc-style (#329561)
+70e758fd18cb03009fb3be691428a92e01dd5f68
```

Comment:
Just drop this? Formatting a single file isn't that problematic

Reply:
Are you sure? The impression I got from #329353 (when I first saw `.git-blame-ignore-revs`) was that it was there to keep more useful information available in git blame, preventing it from being lost by large, syntactically equivalent changes that touch most of the file.

It seemed to me to be perfectly applicable to reformatting with a new formatter, since about %80 of the file has been changed. Is my evaluation incorrect? Is there a reason why I shouldn't change `.git-blame-ignore-revs`?